mod-pubsub-server/src/main/java/org/folio/services/impl/KafkaConsumerServiceImpl.java

```diff
@@ -108,6 +108,7 @@ public KafkaConsumerServiceImpl(@Autowired Vertx vertx,
           messagingModuleList
             .forEach(subscriber -> doRequest(event.getEventPayload(), subscriber.getSubscriberCallback(), HttpMethod.POST, params)
               .setHandler(getEventDeliveredHandler(event, params.getTenantId(), subscriber)));
+          LOGGER.debug("Event with id: {} delivered", event.getId());
```

Comment:
Please place this log should be in the deliveredHandler

Reply:
Also, I think that there is no need to add new log for the delivering event because we already have it in "getEventDeliveredHandler"-method: `LOGGER.info("Delivered event {} to {}", event.getId(), subscriber.getSubscriberCallback());`

Reply:
just make the logger level 'debug' then

Reply:
Ok